examples/pipelines/guestbook.yaml

```diff
@@ -22,10 +22,10 @@ spec:
               name: build-push
           inputSourceBindings:
               - name: workspace
-                key: redis-docker          # TODO Add Commit SHA
+                key: redis-docker          
           outputSourceBindings:
-              - name: registry
-                key: stagingRegistry
+              - name: redisImage
+                key: stagingImage
```

Comment:
i think something is a bit odd here, so this seems to be saying:

1. The Task `build-push` has an output called `redisImage`
2. For that output, use the Resource called `stagingImage`

Looking [at the Task](https://github.com/knative/build-pipeline/blob/b499a94caa9cea3c7337e4922c5c145775a9b7f1/examples/build_task.yaml):

![image](https://user-images.githubusercontent.com/432502/46223522-7aab4700-c308-11e8-8fab-e33d9a63edbe.png)

It looks like the image is actually called `builtImage`, so I think this section should be:

```yaml
outputSourceBindings:
   - name: builtImage
     key: stagingImage
```